geoh5py/shared/merging/drape_model.py

```diff
@@ -139,32 +139,30 @@ def merge_data(
     ):
         super().merge_data(out_entity, input_entities)
 
-        # prepare an empty nan array
-        values: np.ndarray = np.empty(out_entity.n_cells)
-        values[:] = np.nan
+        ind_map = []
+        data_count = 0
+        ghost_count = 0
+        n_values = np.sum([input_entity.n_cells for input_entity in input_entities])
```

Comment:
What about :

`n_values = output_entity.n_cells - (len(input_entities)-1)*2`?
more efficient than a loop no?

Reply:
I am not getting the math here. `n_cells` is all the combined cells

Reply:
yep, and `output.n_cells = sum(*inputs.n_cells) + (len(inputs)-1)*2

Reply:
ok sure yep, both do the same. I think it's a bit harder to read, but if you prefer that.